server/controllers/fast-food.js

```diff
@@ -0,0 +1,21 @@
+import { allFastFoodData } from '../models/fast-food-data';
+
+/**
+ * Represents a Fast Food.
+ */
+class FastFood {
+    /**
+     * This function gets all fastfood
+     * @param {object} req - the request file.
+     * @param {object} res - The response file.
+     * @returns {object} Returns the status and diary information.
+     */
+    allFastFood(req, res) {
+        if (!Object.keys(allFastFoodData.fastFood).length) {
```

Comment:
This should not return a 404. A 404 error tells the client-side that the resource was not found, whereas in this case, the resource was found but empty. In essence, this was a successful operation and shouldn't be a 404 error.